benchmark/src/real_datast.R

```diff
@@ -0,0 +1,42 @@
+source("src/transformations/transformation_helper.R")
+file = readRDS("realdataset,rds") #require both count matrix and size factor vector
+
+UMI = file@assays$data$counts
+sf = file$total_counts
```

Comment:
Are these actually the total counts per cell? The size factors should be scaled such that their mean is 1. 
You can achieve this by adding:
```r
sf <- sf / mean(sf)
```

Reply:
Thanks for your comments. I have fixed it.

But I think a more common understand for size factor or sequecing depth is the sum of total counts for per cell. See sctransform and CS-CORE.
![image](https://user-images.githubusercontent.com/43333475/235660809-e4ddd77e-162b-4a8f-a1df-51ed31dce068.png)
![image](https://user-images.githubusercontent.com/43333475/235661017-a8df929c-21b1-45c6-a293-afbf7257ee73.png)

Reply:
For the log transformation, it is important how you normalize the size factors. See the discussion around eq. (3) for more details.

Reply:
Ok, thanks a lot. I will take a look at it and resubmit my codes.